base/exports.jl

```diff
@@ -248,6 +248,7 @@ export
     At_mul_Bt!,
     At_rdiv_B,
     At_rdiv_Bt,
+    postfixapostrophize,
```

Comment:
Does this really need to be `export`ed?

Reply:
Seems so? Otherwise one must qualify or `import` it broadly?